toc-bar.user.js

```diff
@@ -628,6 +694,14 @@ a.toc-link {
         this.toggle()
         GM_setValue('tocbar-hidden', !this.visible)
       })
+      // toggle keyboard shortcut: alt+t
+      let tocBar = this;
+      document.addEventListener('keydown', function(e) {
```

Comment:
I'm afraid this is too intrusive for the user.
Can we use some of GM or violent monkey's shortcut feature (if there is any) so the user can see and change this shortcut ?